src/main/webapp/js/instructor.js

```diff
@@ -437,25 +437,25 @@ function updateHoverShowPictureEvents(actualLink, resolvedLink) {
 
 /**
  * Selects contents inside an element.
- * @param el
+ * @param elementNode a HTML DOM element object, not a jQuery object
```

Comment:
Would be good to mention the purpose of elementNode in the context of this method. 
i.e. `elementNode The HTML DOM element to select contents from.`